src/app/shell/workshop-details/side-menu/contacts/contacts.component.html

```diff
@@ -22,7 +23,9 @@ <h4>КОНТАКТИ</h4>
     <a [href]="workshop.website">{{workshop.website}}</a>
   </p>
   <div fxLayout="row" fxLayoutAlign="space-between center">
-    <a [routerLink]="''" class="link"><img src="assets/icons/inst.png" alt=""></a>
-    <a [routerLink]="''" class="link"><img src="assets/icons/facebook.png" alt=""></a>
+    <a *ngIf="workshop.instagram" href="{{workshop.instagram}}" class="link"><img src="assets/icons/inst.png"
```

Comment:
if social networks links are not required, we can disable icons if links are not saved